### PR TITLE
III-4516 Document endpoint to copy event

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Documentation for `POST /events/{eventId}/copies`, as an internal endpoint

---

Ticket: https://jira.uitdatabank.be/browse/III-4516

### CODE REVIEW

File: reference/entry.json [hunk 1]
```diff
@@ -1416,6 +1416,225 @@
...
+        },
+        "security": [
+          {
+            "USER_ACCESS_TOKEN": []
+          },
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          }
+        ],
+        "description": "Creates a new event based on all the properties of an existing event with the given `eventId`. Only the calendar information will be completely replaced with new one, which has to be included in the request body.\n\nThe schema of the request body is the same as the one for the `PUT /events/{eventId}/calendar` endpoint.",
```
Comment (LucWollants) [inline reference/entry.json:1634]: Maybe a very small typo : replace `with new one` with `with a new one`

**Review by LucWollants — APPROVED** [review]
Very clear documentation.

File: reference/entry.json (hunk 1 above)
Comment (bertramakers) [reply reference/entry.json]: Good catch!

**Comment by bertramakers** [comment]
I'll merge this once the code is ready as well

**Pull request merged**